src/components/ConfirmModal/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { IoMdClose } from 'react-icons/io';
+
+import Button from '../Button';
+import BodyBackdrop from '../BodyBackdrop';
+import { UiMode } from '../ThemeContext';
+import { useThemeClassName } from '../../hooks';
+
+import styles from './styles.css';
+
+export interface ConfirmModalProps {
```

Comment:
If this overlaps with props of Modal component, you need to create the props by extending from props of Modal component.
See Accordion's prop definition.

Reply:
okay !!